docs/content/configuration/csi_ug_config_create_storageclasses.md

```diff
@@ -24,14 +24,23 @@ _<a name=spaceefficiency>**Table:**</a> `SpaceEfficiency` parameter definitions
 
 - The IBM DS8000 Family `pool` value is the pool ID and not the pool name as is used in other storage systems.
 - Be sure that the `pool` value is the name of an existing pool on the storage system.
+- To create a volume with HyperSwap on IBM Spectrum Virtualize storage systems, put a colon (:) between the two pools within the `pool` value. For example:
+  
+  ```
+  pool: demo-pool1:demo-pool2 
+  ```
+   **Important:** The two pools must be from different sites.
+   
+  For more information about the HyperSwap limitations, see [Limitations](content\release_notes\csi_rn_limitations.md) within the release notes and [Compatibility and requirements](content\installation\csi_ug_requirements.md).
```

Comment:
any idea why these links lead to 404 not found error?

Reply:
fixed. it was a linking issue.
DONE

Reply:
I still get [404](https://github.com/IBM/ibm-block-csi-driver/blob/3723e83c1e96c88cc26c7902f332beccb77abac3/docs/content/configuration/..%5Cinstallation%5Ccsi_ug_requirements.md).
any luck with the link scanner integration?

Reply:
no luck with the integration but my slashes were backwards. i checked it in the files themselves (not just locally) and it works now

Reply:
DONE